src/Resources/assets/controllers/datetime_controller.js

```diff
@@ -4,8 +4,10 @@ import easepickStyle from '!!raw-loader!@easepick/bundle/dist/index.css'
 
 export default class extends Controller {
     static values = {
-        lang: String
+        lang: String,
+        format: String
```

Comment:
Nit: Consider adding JSDocs documentation annotations right above these. Even though IDEs will not pick it up, it helps to know we can open the controller and check the values declarations for how these input values are used or what format they expect.